react-frontend/src/components/StyleComponents/headings.js

```diff
@@ -13,6 +13,15 @@ export const Heading = styled.h2.attrs({
   line-height: 1.6;
 `;
 
+export const FAQHeading = styled.h2.attrs({
+  className: 'faq-heading',
+})`
+  font-size: 18px;
```

Comment:
This goes against the typography standard for font sizes. https://developer.gov.bc.ca/Design-System/Typography  
Font sizes for h2 are supposed to be set at 31.1px